index.html

```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
               Bootstrap
               <span> <img src="images/image3.png" alt="bootstrap" /></span>
             </li>
-            <li>Ruby</li>
+            <li class="list-sumbua">Ruby</li>
           </ul>
           <button class="seeproject-btn" type="submit">See Project</button>
         </div>
       </section>
+      <form action="https://formspree.io/f/xqkngpry" method="post">
+        <h1>CONTACT US</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
```

Comment:
- [ ] These elements should be outside the `form` tag. Is recommendable to only locate form elements inside a form tag